src/Bundler.js

```diff
@@ -68,6 +68,11 @@ class Bundler extends EventEmitter {
       logLevel: typeof options.logLevel === 'number' ? options.logLevel : 3,
       mainFile: this.mainFile,
       hmrPort: options.hmrPort || 0,
+      rootDir: Path.dirname(this.mainFile),
+      sourcemaps:
+        typeof options.sourcemaps === 'boolean'
+          ? options.sourcemaps
+          : !isProduction,
```

Comment:
Why should we disable sourcemaps in production? Isn't that where they are most useful since code is minified? Let's get some feedback on this.

Reply:
i think there are usecases for disabling sourcemaps in production. how about enabling it by default and adding an option `--disable-sourcemaps` - so we can choose at build-time?